src/server/game/DataStores/DBCEnums.h

```diff
@@ -1438,7 +1438,8 @@ enum SummonPropFlags
     SUMMON_PROP_FLAG_UNK19           = 0x00040000,
     SUMMON_PROP_FLAG_UNK20           = 0x00080000,
     SUMMON_PROP_FLAG_UNK21           = 0x00100000,          // Totems
-    SUMMON_PROP_FLAG_COMPANION       = 0x00200000
+    SUMMON_PROP_FLAG_COMPANION       = 0x00200000,
+    SUMMON_PROP_FLAG_UNK22           = 0x04000000
```

Comment:
unrelated?

Reply:
yeah it's not related, I just found it while looking for some stuff and I thought it would be better to have it.

Is it fine, or do you prefer it in another PR? or not at all?